### PR TITLE
[10.0][MIGRATION] Migrated module 'account_fiscal_year' to V10

### PR DESCRIPTION
**Pull request by espo-tony**

### CODE REVIEW

**Comment by espo-tony** [comment]
The Travis test seems to fail because the module date_range, from which this module depends, is not installable yet for V10.
Please, check also OCA/server-tools#567

File: account_fiscal_year/models/res_company.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # Author: Damien Crier
 # Copyright 2016 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-from openerp import models, fields, api
-from openerp.tools import (DEFAULT_SERVER_DATETIME_FORMAT)
+from odoo import models, fields, api
+from odoo.tools.misc import DEFAULT_SERVER_DATETIME_FORMAT
```
Comment (pedrobaeza) [inline account_fiscal_year/models/res_company.py]: You can use fields.Datetime.to_string() instead

File: account_fiscal_year/models/date_range_type.py [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
 
-from openerp import models, fields, api, _, exceptions
+from odoo import models, fields, api, _, exceptions
 
 
 class DateRangeType(models.Model):
     _inherit = "date.range.type"
 
-    fiscal_year = fields.Boolean(string='Is fiscal year ?', default=False)
+    fiscal_year = fields.Boolean(string='Is fiscal year?', default=False)
```
Comment (pedrobaeza) [inline account_fiscal_year/models/date_range_type.py:12]: This change should be reflected in the existing translations.

File: account_fiscal_year/models/res_company.py (hunk 1 above)
Comment (espo-tony) [reply account_fiscal_year/models/res_company.py]: @pedrobaeza 
Thanks for the hint

**Comment by sbidoul** [comment]
@pedrobaeza @gurneyalex do you know why runbot does not run on this PR? The webhook seems to work but the branch does not seem to be on runbot.

**Comment by pedrobaeza** [comment]
Runbot is stuck. We need to restart it.

**Review by alexis-via — APPROVED** [review]
Thanks for this port :+1:

**Comment by gurneyalex** [comment]
I'm on it. Postgresql is having trouble, I'm rebooting the runbots (sorry for the lost instances...)

**Comment by sbidoul** [comment]
@espo-tony can you rebase/force push to trigger runbot?

**Comment by adrienpeiffer** [comment]
I restarted runbot build

**Review by adrienpeiffer — APPROVED** [review]
:+1: Code review and functional test locally

**Pull request merged**